articles/quickstart/native/ios-objc/07-linking-accounts.md

```diff
@@ -79,9 +53,9 @@ NSString *otherUserToken = ... // the idToken from the account you want to link
 }];
 ```
 
-## Retrieve Linked Accounts
+## Retrieve the Linked Accounts
```

Comment:
Retrieving Linked

Reply:
According to the style guide, headings should be in imperative mood as it reads simpler. "The" seems arbitrary, on one hand, it can be omitted in headings, on the other, you retrieve specified accounts (the ones that you linked in the previous step) :)